### ISSUE TITLE
oci_core_ipsec_connection_tunnel_management.shared_secret should be marked "sensitive"

### ISSUE THREAD

**Issue by jeliker** [oracle/terraform-provider-oci#1675]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version
```
Terraform v1.3.1
on darwin_amd64
+ provider registry.terraform.io/hashicorp/oci v4.95.0
```

### Affected Resource(s)
affected_resources = oci_core_ipsec_connection_tunnel_management

### Terraform Configuration Files

```hcl
+ resource "oci_core_ipsec_connection_tunnel_management" "the_tunnel" {
     + compartment_id          = (known after apply)
     + cpe_ip                  = (known after apply)
     + display_name            = "tunnel1"

. . .

     + shared_secret           = "THISSHOULDBEMARKEDSENSITIVENOTSHOWINGTHEWORLDMYPSK"

. . .

```

### Expected Behavior
The shared_secret attribute should be marked "sensitive" so that the PSK is not visible

### Actual Behavior
The shared_secret attribute is not marked "sensitive" so it instead displays in stdout

### Steps to Reproduce

1. `terraform apply`

**Comment by iejones** [comment]
The shared_secret should be marked sensitive. I am making this change. While this update is simple, it is bundled with other bug fixes for this terraform resource that will test and release together.

**Comment by iejones** [comment]
A bug fix including the change to make shared_secret sensitive will be released on November 15th, 2023

**Comment by iejones** [comment]
This fix was released with version [v5.21.0](https://github.com/oracle/terraform-provider-oci/releases/tag/v5.21.0)
